frontend/src/assets/css/variables.scss

```diff
@@ -11,5 +11,8 @@
   --violet-100: #e5e1ff;
   --pink-200: #e1a7d5;
   --red-300: #d51a52;
+  --black-alpha-0: #00000000;
```

Comment:
let's use transparent keyword instead, where it's needed, because usually it's not the color that we will need to change
```suggestion
```

Reply:
done